frame/support/src/weights.rs

```diff
@@ -242,6 +242,30 @@ impl Default for DispatchClass {
 	}
 }
 
+/// Primitives related to priority management of Frame.
+pub mod priority {
+	/// The starting point of all Operational transactions. 3/4 of u64::max_value().
+	pub const LIMIT: u64 = 13_835_058_055_282_163_711_u64;
+
+	/// Wrapper for priority of different dispatch classes.
+	///
+	/// This only makes sure that any value created for the operational dispatch class is
+	/// incremented by [`LIMIT`].
+	pub enum FrameTransactionPriority {
+		Normal(u64),
+		Operational(u64),
+	}
+
+	impl Into<u64> for FrameTransactionPriority {
```

Comment:
Could be `From<FrameTransactionPriority> for u64` which is more idiomatic (you get `Into` automatically).

Reply:
So, I am not ashamed to say that I didn't know this was an exception to the rule that reads: in `impl X for Y` at least one of `X` or `Y` must be items in the local crate :D